src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
+    <div className="row">
+      <Button buttonName="AC">AC</Button>
+      <Button buttonName="+/-">+/-</Button>
+      <Button buttonName="%">%</Button>
+      <Button buttonName="÷">÷</Button>
+    </div>
+    <div className="row">
+      <Button buttonName="7">7</Button>
+      <Button buttonName="8">8</Button>
+      <Button buttonName="9">9</Button>
+      <Button buttonName="X">X</Button>
+    </div>
+    <div className="row">
+      <Button buttonName="4">4</Button>
+      <Button buttonName="5">5</Button>
+      <Button buttonName="6">6</Button>
+      <Button buttonName="-">-</Button>
+    </div>
+    <div className="row">
+      <Button buttonName="1">1</Button>
+      <Button buttonName="2">2</Button>
```

Comment:
We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

Reply:
I'll work on this.